api/tests/unit/domain/validators/schooling-registration-validator_test.js

```diff
@@ -69,14 +69,39 @@ describe('Unit | Domain | Schooling Registration validator', () => {
       });
     });
 
-    context('birthCountryCode', () => {
+    context.only('birthCountryCode', () => {
```

Comment:
Attention un `only` à supprimer
```suggestion
    context('birthCountryCode', () => {
```

Reply:
oups :)